.circleci/config.yml

```diff
@@ -57,7 +73,7 @@ jobs:
            working_directory: ~/build
            command: |
              if (git log -1 --pretty=%s | grep Merge*) && (! git log -1 --pretty=%b | grep REL:) ; then
-             github_changelog_generator --user bids-standard --project bids-specification --token ${CHANGE_TOKEN} --output ~/build/CHANGES.md --base ~/build/src/pregh-changes.md --header-label Changelog --no-issues --no-issues-wo-labels --no-filter-by-milestone --no-compare-link --pr-label ""
+             github_changelog_generator --user bids-standard --project bids-specification --token ${CHANGE_TOKEN} --output ~/build/CHANGES.md --base ~/build/src/pregh-changes.md --header-label "# Changelog" --no-issues --no-issues-wo-labels --no-filter-by-milestone --no-compare-link --pr-label ""
```

Comment:
I am wondering why this line is highlighted as a diff.

In `master`, we should have the exact same line: https://github.com/bids-standard/bids-specification/blob/6bd2ad596f96706c2be4db8cd6551cf85418edd8/.circleci/config.yml#L60

... so there shouldn't be a change, actually :thinking:

Reply:
I am not sure why that's the case. I remember having synced my fork with the latest commits on master, but perhaps I missed something when cherry picking commits from here https://github.com/bids-standard/bids-specification/pull/400 ?